rma/models/stock_move.py

```diff
@@ -47,9 +42,9 @@ def _action_cancel(self):
         # A stock user could have no RMA permissions, so the ids wouldn't
         # be accessible due to record rules.
         cancelled_moves = self.filtered(lambda r: r.state == "cancel").sudo()
-        cancelled_moves.mapped("rma_receiver_ids").write({"state": "draft"})
-        cancelled_moves.mapped("rma_id").update_received_state()
-        cancelled_moves.mapped("rma_id").update_replaced_state()
+        cancelled_moves.rma_receiver_ids.write({"state": "draft"})
```

Comment:
Please reduce diff with useless changes.

Reply:
mapped is useless and has no reason to stay

Reply:
Correct, but still the same reasoning: we don't want a big diff for something unrelated of the goal of the PR. They work the same having `mapped` or not, so that's why it's useless. Do that in another PR doing a refactor if you want.